APE0.rst

```diff
@@ -0,0 +1,393 @@
+outside Voting Members and the Coordination Committee unless the nominee becomes
+a Voting Member.
+
+The procedure for voting to add new Voting Members is:
+
+* The Coordination Committee and Ombudsperson receive each nomination, check
+  that it is factually accurate, that the nominated person accepts the
+  nomination, and that their record of community activity adheres to the Code of
+  Conduct.
+* Once a nomination is accepted by the Coordination Committee, the Coordination
+  Committee will put it to a vote among active Voting Members. Votes for adding
+  voting members will be carried out at most twice a year to avoid too many
+  elections. The voting dates will be announced with at least four weeks notice,
+  and additional nominations will be accepted until two weeks before the
+  election date. The voting will be open for at least two weeks. To successfully
+  gain an appointment as a Voting Member, the nominee must receive at least four
+  positive votes, as long as that constitutes a majority of those voting.
+* The candidate will be informed promptly at the close of voting by a
+  Coordination Committee member. If the vote is not affirmative, the
+  Coordination Committee will provide feedback to the nominee.
+
+At least one round of voting for adding voting members should be carried out
+before the initial election of the Coordination Committee.
+
+Term and Active/Emeritus Status
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Voting Members have no term or term limits. Voting Members who have stopped
+contributing are encouraged to declare themselves as *emeritus*. Those who have
+not made any significant contribution for two years may be asked to move
+themselves to the *emeritus* category by the Coordination Committee. If no
+response is received, the Coordination Committee may automatically change a
+Voting Member’s status to *emeritus*. To record and honor their contributions,
+*emeritus* Voting Members will continue to be listed. *Emeritus* Voting Members
+are not able to participate in votes.
+
+Removing Voting Members
+^^^^^^^^^^^^^^^^^^^^^^^
+In exceptional circumstances, it may be necessary to remove someone from the
+Voting Members against their will. A vote must be held to remove a Voting
+Member. Such a vote is triggered by a motion made by an active Voting Member,
+which must be seconded by an additional Voting Member. The vote must conclude
+no more than three months after the motion is seconded. Removal requires
+approval by two-thirds of all active Voting Members at the time the motion is
+made. The motion, second, and vote will be held in private. Removal under this
+provision will be reflected by updating the list of Voting Members.
+
+It may be necessary for the Ombudsperson and the Coordination Committee to
+remove a Voting Member for violations of the Code of Conduct. In this case, the
+Coordination Committee and Ombudsperson will work together to make this
+decision.
```

Comment:
I have trouble seeing under what circumstances voting members (previous paragraph) might want to remove a members's **vote** (as opposed to their named role, which they might not be carrying out diligently) **other than** a conduct violation.

Reply:
The WG did discuss this, and the consensus there was that this is part of what required the "In exceptional circumstances," clause at the top.  That is, it is indeed hard to be sure what exactly might be cause, but there was agreement that isn't a reason to make it impossible.  The voting process for removal is really meant to be a check in the other direction: if someone things there was a violation that maybe isn't quite covered by the CoC and hence the CoCo/ombudsperson don't feel they can inercede, the members themselves can do so.  That makes it so that the voting members are essentially self-governing because they have control over their own membership, while still allowing the CoCo/ombudsperson the ability to react faster than the voting membership when necessary.

It's plausible that in less egregious cases the CoCo/ombudsperson would likely follow the voting process *anyway*, since they're beholden to the voting members anyway :shrug:

At any rate, while the above hopefully clears things up more, the point is that this was discussed quite a bit before so I don't think we can reverse it at this late of a date.  But as @astrofrog pointed out, this is meant to be a living document!